biohackrxiv-paper-2022/paper-20/paper.md

```diff
@@ -0,0 +1,223 @@
+
+In order to focus our work during the biohackathon we decided to first identify the roles involved in running a Federated EGA node:
+
+* **Node Representative** - aims to understand what services need to be adjusted for their node so that they can run the federated EGA in their infrastructure;
+* **Developer** - needs documentation about the starting points of the code base, in order to start reading, as well as a demo stack of the services;
+* **Operator** - requires a technical checklist to see if the node is technically ready to join federated EGA
+* **Helpdesk** - wants to see the files a submitter has submitted, in order to be able to assist in the submission process
+* **Submitter** - would need to submit data to a Federated EGA node.
+
+Our attention was towards *Developers* and *Submitters* with the starter pack, while the documentation we produced aimed to help *Node Representative* and *Operator* roles.
+
+Given that the *Submitter* role covers a broader spectrum of questions that such a role might have we decided to focus on the encryption of files, which is a requirement for a file to be correctly submitted.
+We underestood that an easy tool that can assist me to encrypt the files (using Crypt4GH [@crypt4gh]) so we aimed to identify and document how such a tool could be utilized.
+
+## FEGA Technical Requirements
+
+We summarized the technical requirements for joining the federation in the following table:
+
+|  # |           Component           |                                                                                                                          Description                                                                                                                          |   Link to  |
+|:--:|:-----------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:----------:|
+| 1  | URL                           | Official website for documentation                                                                                                                                                                                                                            |            |
+| 2  | Credentials                   | For connecting to Central EGA there are 2 types of credentials required:  User API for identifying and validating in Submitter Inbox the cEGA accounts cEGA MQ credentials for connecting FEGA Node MQ                                                        |            |
+| 3  | FEGA Node Encryption key pair | A Crypt4GH key pair generated by the FEGA  Node, and the public is shared with the Submitter so that files are sent  encrypted for the FEGA Node                                                                                                              |            |
```

Comment:
```suggestion
| 3  | FEGA Node Encryption key pair | A Crypt4GH key pair generated by the FEGA  Node, the public is shared with the Submitter so that files are sent  encrypted for the FEGA Node                                                                                                              |            |
```